tests/tests.bats

```diff
@@ -141,24 +141,6 @@ test_install_one_version()
     popd || exit 1
 }
 
-# First real release made this way
-@test "tarball_multi 2.2.2" {
-    local ver=v2.2.2
-    test_init
-
-    get_release "$ver"/sof-bin-"$ver".tar.gz
-
-    "$TOP_DIR"/tarball_multi_releases.bash -g "$ver"     \
-              v2.2.x/sof-v2.2 v2.2.x/tools-v2.2          \
-              v2.2.x/sof-tplg-v2.2.1/ v2.2.x/sof-"$ver"
-    tar xf sof-bin-"$ver".tar.gz
-
-    diff -qr "$EXTR_REFS"/sof-bin-"$ver"  "$(pwd)/sof-bin-$ver"/
-
-    popd || exit 1
-}
-
-
 # You MUST call popd at the end
```

Comment:
It's a bit tricky and I felt your pain but it's not required to throw away or even modify this test. Do this instead:
- don't delete the `sof-v2.2.2` directory`
- create a new, empty file `sof-v2.2.2/placeholder` so git clone still creates the directory
- make this small change instead:

```diff
--- a/tests/compare_signed.bats
+++ b/tests/compare_signed.bats
@@ -56,16 +56,22 @@ test_init()
 # shellcheck disable=SC2030
 @test "compare signed unsigned v2.*/sof-v*" {
     test_init
 
     local sofv
     cd "$TOP_DIR"
     for sofv in v2.*/sof-v*; do
 
+        case "$sofv" in
+            # v2.2.2 has been folded back into v2.2 and is now empty.
+            *v2.2.x/sof-v2.2.2)
+                continue;;
+        esac
+
         run_compare_signed "$sofv"
 
         case "$sofv" in
 
             # special cases

```

Reply:
@marc-hb Thank you! I think this will solve my open. I will update the PR.

Reply:
@marc-hb @lgirdwood PR updated and all checks passed now :-) 
 
tests/static_refs/multi-v2.1.1a/manifest.txt and sha256sum.txt are updated for RPL and ADL-N in 2 commits separately.